2024-02-04 18:56:24,781:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-02-04 18:56:24,781:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-02-04 18:56:24,781:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'marcopac.363.d@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'solidyellow.ddns.net']
2024-02-04 18:56:24,781:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-02-04 18:56:24,940:DEBUG:certbot._internal.log:Root logging level set at 30
2024-02-04 18:56:24,946:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-02-04 18:56:24,947:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7fa965b4f730>
Prep: True
2024-02-04 18:56:24,948:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7fa965b4f730> and installer None
2024-02-04 18:56:24,948:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-02-04 18:56:24,996:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1553477407', new_authzr_uri=None, terms_of_service=None), 633159e3872f852a1e2e97f50627750e, Meta(creation_dt=datetime.datetime(2024, 2, 4, 18, 49, 54, tzinfo=<UTC>), creation_host='b0f1c33b1017', register_to_eff=None))>
2024-02-04 18:56:24,997:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-02-04 18:56:24,999:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-02-04 18:56:25,551:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-02-04 18:56:25,553:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 Feb 2024 18:56:25 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "E3Jle8Sr490": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-02-04 18:56:25,556:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for solidyellow.ddns.net
2024-02-04 18:56:25,560:DEBUG:acme.client:Requesting fresh nonce
2024-02-04 18:56:25,561:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-02-04 18:56:25,740:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-02-04 18:56:25,741:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 Feb 2024 18:56:25 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: kRyagOpUHJjoz03NF4PI-emm6MA2yet5XYG9zu5OKLW-dWMU83M
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-02-04 18:56:25,742:DEBUG:acme.client:Storing nonce: kRyagOpUHJjoz03NF4PI-emm6MA2yet5XYG9zu5OKLW-dWMU83M
2024-02-04 18:56:25,743:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "solidyellow.ddns.net"\n    }\n  ]\n}'
2024-02-04 18:56:25,745:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MzQ3NzQwNyIsICJub25jZSI6ICJrUnlhZ09wVUhKam96MDNORjRQSS1lbW02TUEyeWV0NVhZRzl6dTVPS0xXLWRXTVU4M00iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "gFuyUL4tdrLKtENQjT6w4ZaniCfXOkLsJft7rvci_795swgmxY8ncqId-yadJIdS1-pf4k3tNzqU-1HT9MLaPiqk8GXpNDvr2vQjYURlcJ4gMejn7IJFYvBN5FxSWATXaBYfdqymWWdFEAf9DOYZWJeGJIf0Qm_OE4LWIopIUtdMuHvOBfF4XeK0kXejsmhQRRaamxKb2DWQXZJ_RuWKVmOl_qAyqhas_HM2qZqr05EpF5W-IoPIzuws4yh1Oh-3v_BFShP2FqhG3jFGJwL8OyHcTtInLzYKCaCr727FSMQ88U5ijdv6617iPWjxsbk02dOqZM4Jf3d_YRB8hZy_Yg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInNvbGlkeWVsbG93LmRkbnMubmV0IgogICAgfQogIF0KfQ"
}
2024-02-04 18:56:25,952:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 346
2024-02-04 18:56:25,953:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 04 Feb 2024 18:56:25 GMT
Content-Type: application/json
Content-Length: 346
Connection: keep-alive
Boulder-Requester: 1553477407
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1553477407/241868077947
Replay-Nonce: 2ksh1KUsLVi9qslv3ppFprM8RYG74WzUP1Z7Squ5PrLHcjXqtx0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-02-11T18:56:25Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "solidyellow.ddns.net"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/311604930327"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1553477407/241868077947"
}
2024-02-04 18:56:25,954:DEBUG:acme.client:Storing nonce: 2ksh1KUsLVi9qslv3ppFprM8RYG74WzUP1Z7Squ5PrLHcjXqtx0
2024-02-04 18:56:25,955:DEBUG:acme.client:JWS payload:
b''
2024-02-04 18:56:25,956:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/311604930327:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MzQ3NzQwNyIsICJub25jZSI6ICIya3NoMUtVc0xWaTlxc2x2M3BwRnByTThSWUc3NFd6VVAxWjdTcXU1UHJMSGNqWHF0eDAiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMTYwNDkzMDMyNyJ9",
  "signature": "Cd8ulIUr5kxClu-oN-5S1RZ0NEGtsC4v7heI92-Xub8lxQ7IltygmApIMb2a46Dtm9aOh27XTjBipXIN3VK00ePqp64lstAopICYiqJzGI733cKz_miMgdGCtmHMWqOG1iCgEJl5XqYpY4Hm6M46PBQSD1eVRnWtIrc2RtDGOIw_3pcq10eeXoJsdHYr2PZXf1-Z56NkmFj0hqwkhg0xTM0bA7Usy-eMPlimgqNf0GX8cUB9JzgT-VWEtHXxT1oZMv_wy2qeDuBVwRg_6LwtJzWEUpORj_NUtptwBbEOpAxssTx1y-uwW0OjK10RIoZPx3tJWrznaGRinB2RxMWpfg",
  "payload": ""
}
2024-02-04 18:56:26,140:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/311604930327 HTTP/1.1" 200 804
2024-02-04 18:56:26,141:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 Feb 2024 18:56:26 GMT
Content-Type: application/json
Content-Length: 804
Connection: keep-alive
Boulder-Requester: 1553477407
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsiNdfhw3A4nf9Z9Ts636Y1TzGDuru6Ks78OKYaiegBXw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "solidyellow.ddns.net"
  },
  "status": "pending",
  "expires": "2024-02-11T18:56:25Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/HDLcOQ",
      "token": "YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/ewig_A",
      "token": "YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/PRaaCA",
      "token": "YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s"
    }
  ]
}
2024-02-04 18:56:26,142:DEBUG:acme.client:Storing nonce: 2ksh1KUsiNdfhw3A4nf9Z9Ts636Y1TzGDuru6Ks78OKYaiegBXw
2024-02-04 18:56:26,142:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-02-04 18:56:26,143:INFO:certbot._internal.auth_handler:http-01 challenge for solidyellow.ddns.net
2024-02-04 18:56:26,144:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2024-02-04 18:56:26,145:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2024-02-04 18:56:26,151:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s
2024-02-04 18:56:26,154:DEBUG:acme.client:JWS payload:
b'{}'
2024-02-04 18:56:26,156:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/HDLcOQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MzQ3NzQwNyIsICJub25jZSI6ICIya3NoMUtVc2lOZGZodzNBNG5mOVo5VHM2MzZZMVR6R0R1cnU2S3M3OE9LWWFpZWdCWHciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMxMTYwNDkzMDMyNy9IRExjT1EifQ",
  "signature": "B5RW64M-4105GmSZ6odAsx2E_8vMtjkbK0vFbmP8SNNJWdYmsc3cDPJEtBK_nDYV8IF72ozHsMpmOBWAejlN1c_zNE9Fu259NZR3vLB0u-cX_bLZRSuw1QaoPsrK_h9DH43AWbD9REDALQzlf0uQJNZ4L8u5sbRapfgGXOkPKJSH92npCjGECJm0YibiTfoV3gaQ7kCDptC5mNxKjqk1FncGfvPVNaE7z78K5O7re4hnQ8XJiJ8JhvEoLgrCD21ilXYX3f3gMJS7P3GAy7ZqzyYMuTThJfCRpJTLvUPcoB_pY-tN1gFQfxt8pGEtnB78m9msMICBNHRQ0ysMQFy7DA",
  "payload": "e30"
}
2024-02-04 18:56:26,341:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/311604930327/HDLcOQ HTTP/1.1" 200 187
2024-02-04 18:56:26,342:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 Feb 2024 18:56:26 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1553477407
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/311604930327>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/HDLcOQ
Replay-Nonce: kRyagOpUNJRx9rop7GVpWFUdEk5byCIu_ys9gs2An9DOfdLqPRs
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/HDLcOQ",
  "token": "YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s"
}
2024-02-04 18:56:26,343:DEBUG:acme.client:Storing nonce: kRyagOpUNJRx9rop7GVpWFUdEk5byCIu_ys9gs2An9DOfdLqPRs
2024-02-04 18:56:26,343:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-02-04 18:56:27,345:DEBUG:acme.client:JWS payload:
b''
2024-02-04 18:56:27,346:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/311604930327:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MzQ3NzQwNyIsICJub25jZSI6ICJrUnlhZ09wVU5KUng5cm9wN0dWcFdGVWRFazVieUNJdV95czlnczJBbjlET2ZkTHFQUnMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMTYwNDkzMDMyNyJ9",
  "signature": "BtzfkluKk_iQ44VNc4YgkKMS1LwKK9kdAcFrz1n0HHmlrc60L63nkPmjaHiVZvY_d6cQ3FR6LAyIM2rro3w1TFNugDNxzoj8TyhVUNhCk0ELLO9dTjPBJz6kuelFQly0A17cSFsSiMylnHhTmLKpSFoCB8v1_sX5lKuj-x1HG_wjHfFG-U2ajZstGmtT58F4c95bQVOFm68qadzyrLKky9B49cdYDyiU9HX5UGlPySfAJfVYgSDw9dc7t_Gc4KpD7l7_7u4Bqzx0Oc33y44grf1w_kM0rgJb7jFS5HYptHMR_1BSYQzMo4AK_RFDpT1MdLEdjAGMPYQkQlMkXbc4EQ",
  "payload": ""
}
2024-02-04 18:56:27,530:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/311604930327 HTTP/1.1" 200 1048
2024-02-04 18:56:27,531:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 Feb 2024 18:56:27 GMT
Content-Type: application/json
Content-Length: 1048
Connection: keep-alive
Boulder-Requester: 1553477407
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsjV9EViDQs8kaMyh6PVEA-f6SNBODgUFcsbF0rHB3XpI
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "solidyellow.ddns.net"
  },
  "status": "invalid",
  "expires": "2024-02-11T18:56:25Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "80.116.98.219: Invalid response from http://solidyellow.ddns.net/.well-known/acme-challenge/YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s: 404",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/311604930327/HDLcOQ",
      "token": "YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s",
      "validationRecord": [
        {
          "url": "http://solidyellow.ddns.net/.well-known/acme-challenge/YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s",
          "hostname": "solidyellow.ddns.net",
          "port": "80",
          "addressesResolved": [
            "80.116.98.219"
          ],
          "addressUsed": "80.116.98.219"
        }
      ],
      "validated": "2024-02-04T18:56:26Z"
    }
  ]
}
2024-02-04 18:56:27,532:DEBUG:acme.client:Storing nonce: 2ksh1KUsjV9EViDQs8kaMyh6PVEA-f6SNBODgUFcsbF0rHB3XpI
2024-02-04 18:56:27,533:INFO:certbot._internal.auth_handler:Challenge failed for domain solidyellow.ddns.net
2024-02-04 18:56:27,533:INFO:certbot._internal.auth_handler:http-01 challenge for solidyellow.ddns.net
2024-02-04 18:56:27,534:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: solidyellow.ddns.net
  Type:   unauthorized
  Detail: 80.116.98.219: Invalid response from http://solidyellow.ddns.net/.well-known/acme-challenge/YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-02-04 18:56:27,535:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-02-04 18:56:27,536:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-02-04 18:56:27,537:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-02-04 18:56:27,538:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/YNDVEcLgSi-1ACJkUWIixlNuKBmguaZLxL9zIpmD98s
2024-02-04 18:56:27,541:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-02-04 18:56:27,542:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-02-04 18:56:27,543:ERROR:certbot._internal.log:Some challenges have failed.
